databricks/koalas/frame.py

```diff
@@ -881,8 +881,47 @@ def unique(self):
         sdf = self._sdf
         return DataFrame(spark.DataFrame(sdf._jdf.distinct(), sdf.sql_ctx), self._metadata.copy())
 
-    @derived_from(pd.DataFrame)
     def drop(self, labels, axis=0, errors='raise'):
+        """
+        Drop specified labels from columns.
+
+        Remove columns by specifying label names and axis=1.
+        Removing rows is yet to be implemented.
+
+        Parameters
+        ----------
+        labels : single label or list-like
+            Column labels to drop.
+        axis : {0 or 'index', 1 or 'columns'}, default 0
```

Comment:
say something about only axis = 1 is supported at the moment.

Reply:
Added the information.

Reply:
I think we should fix this doc.